### PR TITLE
fix(frontend): easy to read and consistent with G*Power

### PR DESCRIPTION
**Pull request by storopoli**
Changes:

- Makes the text larger and easy to read
- Menu positions are consistent with G*Power
- HTML language server (the default from nvim/helix/vscode) autoformatted the HTML and CSS
- GitHub URL updated to new repo/org name

---

G*Power:
image

---

Before (desktop):
image

After (desktop):
image

---

Before (mobile):
image

After (mobile):
image

### CODE REVIEW

**Comment by storopoli** [comment]
The only thing is that now the `Statistical test` menu in mobile is truncated.

File: frontend.js [hunk 1]
```diff
@@ -92,7 +92,7 @@ function removeAllTableRows(table) {
     }
 }
 
-/** Add an option to a table. */
+// Add an option to a table.
```
Comment (rikhuijzer) [inline frontend.js:95]: The default way to format Javascript docstrings is with `/** ... */`

**Review by rikhuijzer — COMMENTED** [review]
> Menu positions are consistent with G*Power

What changed to make things more consistent?

**Comment by rikhuijzer** [comment]
> The only thing is that now the `Statistical test` menu in mobile is truncated.

Looks still clear so I'm okay with that

File: frontend.js (hunk 1 above)
Comment (storopoli) [reply frontend.js:95]: Oh sorry, I will revert these. I had no idea these were docstrings. Makes sense now.

**Comment by storopoli** [comment]
> What changed to make things more consistent?

The menu positions:

Before:

```
test family | statistical test | type
```

Now:

```
test family | statistical test
type        |
```

**Review by rikhuijzer — APPROVED** [review]

File: frontend.js (hunk 1 above)
Comment (storopoli) [reply frontend.js]: Should be fixed now. I am not touching `frontend.js` here.

**Pull request merged**